hw3/people.data.js

```diff
@@ -1,48 +1,48 @@
 export const SarahMay = {
-  firstName: "Sarah",
-  lastName: "May",
-  birthDate: "13.03.1990",
+  firstName: 'Sarah',
```

Comment:
why have you decided to change the type of ticks in this file?

Reply:
ahaha, maybe It was eslint+prettier  :)